doc/neps/nep-0045-c_style_guide.rst

```diff
@@ -0,0 +1,269 @@
+=================================
+NEP 45 — C Style Guide
+=================================
+
+:Author: Charles Harris <charlesr.harris@gmail.com>
+:Author: Pierre de Buyl <pdebuyl@pdebuyl.be>
+:Author: Matti Picus <matti.picus@gmail.com>
+:Author: Warren Weckesser <warren.weckesser@gmail.com>
+:Author: Parul Aggarwal <parulagg27@gmail.com>
```

Comment:
@charris should be listed as the sole author here. He wrote the whole thing, all other commits to that file are minor tweaks.